### DISCUSSION TITLE
Editable table

### DISCUSSION THREAD

**Discussion by mturoci** [h2oai/wave#652] [Ideas]
via @vopani 

Provide a mechanism to edit table data directly from within table component. E.g. clicking on done icon would edit the underlying data and sync it with qd server. This can be also extended to editable textfields.

cc @lo5

**Comment by mtanco** [comment]
Related, there has been requests for this to be doable on the whole table (multiple rows and columns) at once

**Reply by MaksymSofer** [reply]
I would like to provide some use cases:
1. An Operator (shop manager for example) have CSV that can be uploaded into table, edit the table with possible interaction with DB along the way and save the data to DB or export another format for further use.
2. Data could require "human validation" before visualization (upload data to table -> edit data -> visualize ).
3. Some could use static file for data storage and so we can: read file -> represent in table -> edit table -> save back to the file

**Comment by sakunaharinda** [comment]
Hi, did you guys provide any support for these editable tables?
